### ISSUE TITLE
Error occurring in prompt_for_user_token() (Python 2.7.12)

### ISSUE THREAD

**Issue by clungzta** [plamere/spotipy#210]
Hello,

The following error occurs in the prompt_for_user_token() function when using Spotipy 2.4.3 and 2.4.4.

I am currently running Python version 2.7.12. I've set the Spotify auth environment variables in my bashrc.

```
$python my_playlists.py [USERNAME]
Traceback (most recent call last):
  File "my_playlists.py", line 18, in <module>
    token = util.prompt_for_user_token(username, scope)
  File "/usr/local/lib/python2.7/dist-packages/spotipy/util.py", line 56, in prompt_for_user_token
    token_info = sp_oauth.get_cached_token()
  File "/usr/local/lib/python2.7/dist-packages/spotipy/oauth2.py", line 135, in get_cached_token
    if 'scope' not in token_info or not self._is_scope_subset(self.scope, token_info['scope']):
  File "/usr/local/lib/python2.7/dist-packages/spotipy/oauth2.py", line 156, in _is_scope_subset
    needle_scope = set(needle_scope.split())
AttributeError: 'NoneType' object has no attribute 'split'
```

If I revert back to Spotipy-2.3.8 this error does not occur. (but 2.3.8 is missing some functionality which is available in 2.4.4 that I would like to use)

It would be greatly appreciated if any suggestions/fixes could be provided.

**Comment by HamedMP** [comment]
I solved it with this #issuecomment-317192954

**Comment by gwynnebaer** [comment]
Same solution worked for me.

**Comment by stephanebruckert** [comment]
> If I revert back to Spotipy-2.3.8 this error does not occur. (but 2.3.8 is missing some functionality which is available in 2.4.4 that I would like to use)

There was a bug with 2.4.4 where all the code was actually from a much older version. Please upgrade:

    pip install spotipy --upgrade
